### PR TITLE
add debug mode for non-x64

### PR DESCRIPTION
**Pull request by zhaoyanggh**

### CODE REVIEW

File: deps/CMakeLists.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+IF (TD_ARM_32)
```
Comment (sangshuduo) [inline deps/CMakeLists.txt:1]: I suggest modifying compile option in the release section only and using add_compile_options() to avoid affecting globally.

**Review by sangshuduo — CHANGES_REQUESTED** [review]
Please hold this PR. We should find a solution with minimal effection.

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by afwerar — APPROVED** [review]
LGTM

**Pull request merged**